dashboard/app/models/script.rb

```diff
@@ -1471,6 +1471,20 @@ def get_assessment_script_levels
   end
 
   def get_feedback_for_section(section)
+    rubric_performance_headers = {
+      "performanceLevel1": "Extensive Evidence",
+      "performanceLevel2": "Convincing Evidence",
+      "performanceLevel3": "Limited Evidence",
+      "performanceLevel4": "No Evidence"
+    }
+
+    rubric_performance_json_to_ruby = {
```

Comment:
you don't need quotes around the keys, especially since you are converting the keys `to_sym` before looking them up.